rdt-open-reader/build.gradle

```diff
@@ -28,7 +28,7 @@ android {
         versionCode 12
         versionName "v1.15.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        buildConfigField "int", "MAX_SYNC_RETRIES", '3'
+        buildConfigField "int", "MAX_SYNC_RETRIES", '1'
```

Comment:
I don't think we need to do this since we don't get to the push function if authorization fails.

Reply:
done.